apis/certificatemanager/v1beta1/types.generated.go

```diff
@@ -0,0 +1,340 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+// +kcc:proto=google.cloud.certificatemanager.v1.Certificate
+type Certificate struct {
+	// A user-defined name of the certificate. Certificate names must be unique
+	//  globally and match pattern `projects/*/locations/*/certificates/*`.
```

Comment:
Nit: We seem to add an additional white space at the beginning of the comment lines except for the first line in these generated types. Not sure whether it is intended but it doesn't seem to be a commonly used format of commenting (I think?)

Reply:
The space is gen-ai styles. we can come back and fix it later if needed